### DISCUSSION TITLE
Same proxy address on all the supported chains

### DISCUSSION THREAD

**Discussion by SolcAndMe** [PaulRBerg/prb-proxy#60] [Q&A] [answered]
Given recent events sometimes proxy users make the mistake of sending funds to the address of their proxy account (deployed on e.g. mainnet) to another network on which the proxy hasn't been deployed yet. Given that the Factory is deployed to the same address on every network and Proxy are generally deployed using Create2 - the first proxy the user deploys on every network should have the same address, is that correct?

**Comment by PaulRBerg** [comment, marked as answer]
Good question!

Yes, the user's proxy addresses will be the same on all chains. This is, in fact, one of the major advantages of PRBProxy over DSProxy - namely, that the PRBProxy factory and registry have been deployed with a [deterministic-deployment-proxy](https://github.com/Zoltu/deterministic-deployment-proxy).

I have just tested this by deploying a proxy on BSC and Polygon. You can see that in both cases, the proxy's address is [0x8119cd5237449486FA4577F30b90CFa37aA03Dfc](https://polygonscan.com/address/0x8119cd5237449486FA4577F30b90CFa37aA03Dfc):

- [Polygon tx](https://polygonscan.com/tx/0x5db1acc9fd0ecf54a22dbdb252e69c6e06290692833a6200bc63d208dd1e146f#eventlog)
- [BSC tx](https://bscscan.com/tx/0x8e14c7b4536b90d95208048c93bc586f2a8f24f2a5c7f3ddba01a55027c375f3#eventlog)

Speaking of factories vs registries - for the best user experience, I recommend using the registry instead of the factory.
